sky/utils/schemas.py

```diff
@@ -43,8 +44,13 @@ def _check_not_both_fields_present(field1: str, field2: str):
         {
             # Shorthand to set idle_minutes by directly specifying, e.g.
             #   autostop: 5
-            'type': 'integer',
-            'minimum': 0,
+            'anyOf': [{
+                'type': 'string',
+                'pattern': resources.TIME_PATTERN,
+                'minimum': 0,
+            }, {
+                'type': 'integer',
+            }]
         },
         {
             'type': 'object',
```

Comment:
follow-up (below) - use idle_time rather than idle_minutes so we can write e.g.
```
autostop:
  idle_time: 1d
  down: true
```
Could you create a TODO comment or ticket for this?